internal/app/machined/main.go

```diff
@@ -126,21 +126,25 @@ func main() {
 		handle(err)
 	}
 
-	// Initialize the machine.
-	if err = c.Run(runtime.SequenceInitialize, nil); err != nil {
-		handle(err)
-	}
-
-	// Start event listeners.
+	// Start signal and ACPI listeners.
+	// NB: This MUST be started AFTER the initialize sequence since there are no
+	// pseudo filesystems available until then.
 	go func() {
 		if e := c.ListenForEvents(); e != nil {
 			log.Printf("WARNING: signals and ACPI events will be ignored: %+v", e)
 		}
 	}()
 
+	// Initialize the machine.
+	if err = c.Run(runtime.SequenceInitialize, nil); err != nil {
+		handle(err)
+	}
+
 	// Start the API server.
+	// NB: This MUST be started AFTER the initialize sequence since the system
+	// directories are setup then.
 	go func() {
-		server := &v1alpha1server.Server{
+		s := &v1alpha1server.Server{
```

Comment:
Align with the convention used throughout this file (single letter vars is a thing in go).